cli/src/config/hints.toml

```diff
@@ -0,0 +1,2 @@
+[hints]
+resolving-conflicts = true
```

Comment:
Can you also update `config-schema.json`?

I don't think we'll need detailed documentation per hint key (because it would be tedious to update.) Something like `"aliases"` or `"template-aliases"` should be good.

Reply:
I've done that. But presumably it's best to document the list of hint keys that you can hide somewhere, similar to [git's docs?](https://git-scm.com/docs/git-config#Documentation/git-config.txt-advice).

Reply:
> That's one reason I suggested `Hint: {message} [{key}]` suffix.

I like this idea. I think I'd ultimately want something more structured (so that we can e.g. generate the json schema from the same source of truth), but until then, this would work. Even if the source of truth becomes more structured, printing hints in this way might still make sense.

Yuya, correct me if I'm wrong, but I don't think you meant this to be a blocking commend, did you? I think of this as a plan for the future; once there is more than one hint config, the message can be changed to this form.

Reply:
No blocker. Things have to be fixed are just "string" -> "boolean".